### PR TITLE
intel-ucode: update to 20240813

### LINKED ISSUE #7707: Intel Processor Microcode (intel-ucode) : security update to microcode-20240813
### Affected package (and version)

`intel-ucode` (<20240813)

### CVE ID(s)

CVE-2024-24853, CVE-2024-25939, CVE-2024-24980, CVE-2023-42667, CVE-2023-49141

### Severity

High

### Other security advisory ID(s）

- [2024.3 IPU - SMI Transfer Monitor Advisory](https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-01083.html)
- [3rd Generation Intel® Xeon® Scalable Processor Advisory](https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-01118.html)
- [2024.3 IPU - Intel® Xeon® Processor Advisory](https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-01100.html)
- [Intel® Core™ Ultra Processor Stream Cache Advisory](https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-01038.html)
- [2024.2 IPU - Intel® Processor Stream Cache Advisory](https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-01046.html)

### Description/References

[microcode-20240813 Release](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20240813)

### Patch(es)/Solution(s)

Update to **microcode-20240813**.

### PR DESCRIPTION
**Pull request by ouankou**
Topic Description
-----------------

- intel-ucode: update to 20240813

Package(s) Affected
-------------------

- intel-ucode: 20240813

Security Update?
----------------

No

Build Order
-----------

```
#buildit intel-ucode
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`

### CODE REVIEW

**Comment by ouankou** [comment]
Tested on i9-11950H.
```bash
(ssh)ouankou@ei [ ~ ] $ sudo dmesg | grep microcode
[    1.493354] microcode: Current revision: 0x00000052
[    1.493359] microcode: Updated early from: 0x00000050
```

File: runtime-data/intel-ucode/spec [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VER=20240531
+VER=20240813
 SRCS="tbl::https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/archive/microcode-$VER.tar.gz"
```
Comment (KexyBiscuit) [inline runtime-data/intel-ucode/spec]: Use git source.

**Review by KexyBiscuit — CHANGES_REQUESTED** [review]

File: runtime-data/intel-ucode/spec (hunk 1 above)
Comment (ouankou) [reply runtime-data/intel-ucode/spec]: Done.

**Review by KexyBiscuit — APPROVED** [review]
LGTM.

**Pull request merged**